pegasus/sites.v3/code.org/views/top_two_hoc_tutorials.haml

```diff
@@ -0,0 +1,27 @@
+.col-50
+  .tutorial-tile
+    %a{href: CDO.studio_url("/s/applab-intro/reset")}
```

Comment:
didn't realize the content, not just the layout, is the same as code.org/applab
Would be nice to make a haml partial, but I get we need to ship so non-blocking.